src/routes/console/project-[project]/auth/templates/+page.svelte

```diff
@@ -0,0 +1,223 @@
+            });
+        }
+    }
+</script>
+
+<script lang="ts">
+    import { goto } from '$app/navigation';
+    import { base } from '$app/paths';
+    import { page } from '$app/stores';
+    import { Alert, CardGrid, Collapsible, CollapsibleItem, Heading } from '$lib/components';
+    import { Container } from '$lib/layout';
+    import { sdk } from '$lib/stores/sdk';
+    import { project } from '../../store';
+    import { addNotification } from '$lib/stores/notifications';
+    import { onMount } from 'svelte';
+    import type { PageData } from './$types';
+    import EmailVerificationTemplate from './emailVerificationTemplate.svelte';
+    import EmailMagicUrlTemplate from './emailMagicUrlTemplate.svelte';
+    import EmailRecoveryTemplate from './emailRecoveryTemplate.svelte';
+    import EmailInviteTemplate from './emailInviteTemplate.svelte';
+    import SmsVerificationTemplate from './smsVerificationTemplate.svelte';
+    import SmsLoginTemplate from './smsLoginTemplate.svelte';
+    import { baseEmailTemplate, baseSmsTemplate, emailTemplate, smsTemplate } from './strote';
+    import { Button } from '$lib/elements/forms';
+
+    export let data: PageData;
+    const projectId = $page.params.project;
+
+    let emailOpen = 'verification';
+    $: emailVerificationOpen = emailOpen === 'verification';
+    $: emailMagicSessionOpen = emailOpen === 'magicSession';
+    $: emailResetPassword = emailOpen === 'recovery';
+    $: emailInviteUser = emailOpen === 'invitation';
+
+    let smsOpen = 'verification';
+    $: smsVerificationOpen = smsOpen === 'verification';
+    $: smsLoginOpen = smsOpen === 'login';
+    $: smsInvitationOpen = smsOpen === 'invitation';
+
+    onMount(async () => {
+        openEmail('verification');
+        openSms('verification');
+    });
+
+    async function openEmail(type: string) {
+        type === emailOpen ? (emailOpen = null) : (emailOpen = type);
+        $emailTemplate = await loadEmailTemplate(projectId, type, 'en');
+        $baseEmailTemplate = { ...$emailTemplate };
+    }
+
+    async function openSms(type: string) {
+        type === smsOpen ? (smsOpen = null) : (smsOpen = type);
+        $smsTemplate = await loadSmsTemplate(projectId, type, 'en');
+        $baseSmsTemplate = { ...$smsTemplate };
+    }
+</script>
+
+<Container>
+    <div class="u-flex u-gap-8 u-cross-center">
+        <Heading tag="h2" size="5">Templates</Heading>
+        <div class="tag eyebrow-heading-3">
+            <span class="text u-x-small">Beta</span>
+        </div>
+    </div>
+
+    {#if !$project.smtpEnabled}
+        <div class="u-margin-block-start-24">
+            <Alert
+                isStandalone
+                dismissible
+                type="info"
+                buttons={[
+                    {
+                        name: 'Add SMTP server',
+                        method: () => {
+                            goto(base + '/console/project-' + $project.$id + '/settings/smtp');
```

Comment:
we use template strings in other areas of the code where we [use `goto`](https://github.com/search?q=repo%3Aappwrite%2Fconsole%20goto(&type=code)

Let's use the same pattern here